clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
+created a slideshow presentation to help you run your first meeting.
+[Click here][presentation] to go to it. We'll be referring to this presentation
+throughout the sections below.
+
+1. [Wait for hackers (club members) to arrive (5 minutes)][1]
+2. [Introduction presentation (5 minutes)][2]
+3. [Name game (10 minutes)][3]
+4. [Build your first website workshop (45 minutes)][4]
+5. [Free-form hacking (25 minutes)][5]
+6. [Second meeting reminder][6]
+
+[presentation]: https://docs.google.com/presentation/d/1QhErlJS6rnqu78guPBWuJ1SUyPJKLbxzLNuSN9qVw0c/edit
+[1]: #waiting-for-hackers-to-arrive
+[2]: #introduction-presentation
+[3]: #name-game
+[4]: #first-website-workshop
+[5]: #free-form-hacking
+[6]: #second-meeting-reminder
+
+#### Waiting for Hackers to Arrive
+
+As hackers start to arrive, your goal is to make them feel welcome. We recommend
+doing the following four things to accomplish this:
+
+1. Put the first slide of the previously mentioned presentation on the projector
+2. Put some music on (not too loud, [click here][playlist] for a good pre-made
+   Spotify playlist)
+3. Make people feel welcome as they walk in
+  - Introduce yourself
+  - Ask a couple questions to initiative conversation
+    - "How did you hear about Hack Club?"
+    - "What sorts of things are you interested in learning?"
+  - You want people to feel good about their decision to come to Hack Club
+4. Tell people what will be happening next (ex. "We're going to get the first
+   meeting started in about 10 minutes after people are done trickling in. In
+   the meantime, please make yourself at home and let me know if I can be
+   helpful with anything.")
+
+[playlist]: https://open.spotify.com/user/zachlatta/playlist/16lRCHBANauZr4utynaA2B
+
+#### Introduction Presentation
+
+The purpose of the introduction presentation is to tell members what Hack Club
+is, what you're going to be doing throughout the year, when you meet, and, _most
+importantly_, get them excited about the club.
+
+The beginning of the previously mentioned slideshow presentation contains slides
+to run with the introduction presentation.
+
+#### Name Game
+
+The name game is a simple icebreaker activity that will warm the atmosphere and
+make club members more comfortable around each other. This will be especially
+helpful to freshman, who probably don't know people from the higher grades.
+
+Have the club members gather in large circle. Start by saying your name and a
+mnemonic for it (ex. "Hey everyone, I'm Zach. Zany Zach."). Then have the person
+to your left do the same and then repeat what you said (ex. "I'm Jolly Jane. And
+he is Zany Zach"). Then have the person to their left do the same and then say
+the names and mnemonics for everyone that has gone so far, in order. Continue
+doing this through the whole circle until everyone has gone. This will get
+significantly more difficult (and fun!) as more people go -- if someone gets
+completely stuck, encourage others to help them out.
```

Comment:
I don't think it's worth simplifying but I do think that is kinda broken if there's not video because no matter what you write, I don't think it will be clear enough.